WINDOWS_SETUP.md

```diff
@@ -87,7 +88,7 @@ Next, we need the build tools. The choice we make here will determine the Androi
 
 15. And for debugging, later on, we need the platform-tools: `sdkmanager "platform-tools"`
 
-Now, time for Flutter. Flutter uses the snap package manager, sadly this doesn't work in WSL 2 out of the box because of an issue with systemd at time of writing. So, let's fix it first!
+Now, time for Flutter. Flutter uses the snap package manager, sadly this doesn't work in WSL 2 out of the box because of an issue with system at the time of writing. So, let's fix it first!
```

Comment:
I believe `systemd` might be correct here if it is referring to the software. @roel-de-vries could you please confirm or deny this? 😄

Reply:
It's indeed referring to [systemd](https://en.m.wikipedia.org/wiki/Systemd), so this change shouldn't be needed. 😁

Reply:
ohk.